packages/web/src/components/pages/WebsiteResults.tsx

```diff
@@ -87,9 +111,47 @@ export function WebsiteResultsPage() {
         dispatch(resetScanDisplayOptions({ scanUrl: normalizedUrl }));
       }
     },
-    [dispatch, normalizedUrl]
+    [dispatch, normalizedUrl, selectedDisplayOptions]
+  );
+
+  const handleSortersChange = useCallback(
+    (newSorters: PackageSorter[]) => {
+      if (!normalizedUrl) {
+        return;
+      }
+
+      dispatch(
+        setScanDisplayOptions({
+          scanUrl: normalizedUrl,
+          options: {
+            ...selectedDisplayOptions,
+            packageSorters: newSorters,
+          },
+        })
+      );
+    },
+    [dispatch, normalizedUrl, selectedDisplayOptions]
   );
 
+  const selectedSortField = selectedDisplayOptions.packageSorters[0].by;
+  const selectedSortDirection = selectedDisplayOptions.packageSorters[0].direction;
+
+  const handleSortChange = useCallback(
+    (newSorterName: PackageSortType) => {
+      const sortOrder =
+        newSorterName === selectedSortField && selectedSortDirection === 'DESC' ? 'ASC' : 'DESC';
```

Comment:
What are your opinion on moving this logic into the sorter component itself? So it receives current sorting order as a whole `PackageSorter` and passes back a new one in a cb?

Reply:
I can see that if sorter would be just one component. In our case we got two - desktop and mobile one. Would duplication of the handler locally on both components be worth it in this case?

Reply:
Valid point, LGTM for now.